README.md

```diff
@@ -24,7 +24,8 @@ found
 ## Prerequisites
 
 - A fully updated Steam version of Doom Eternal
-- Mono: See [here](https://wiki.archlinux.org/index.php/Mono) for more information.
+- .NET Core: See [here](https://wiki.archlinux.org/index.php/.NET_Core) for more
```

Comment:
I'm not sure what happened here. Maybe you weren't working from the latest version but the dependency should be `Mono` instead of `.NET Core`. Just set this part back to what it was 

```
Mono: See [here](https://wiki.archlinux.org/index.php/Mono) for more information.
```

Reply:
Oh yes, sorry, this was an accident